src/univariate/continuous/loguniform.jl

```diff
@@ -0,0 +1,81 @@
+"""
+    LogUniform(a,b)
+
+A positive random variable `X` is log-uniformly with parameters `a` and `b` if the logarithm of `X` is `Uniform(log(a), log(b))`.
+The *log uniform* distribution is also known as *reciprocal distribution*.
+```julia
+LogUniform(1,10)
+```
+External links
+
+* [Log uniform distribution on Wikipedia](https://en.wikipedia.org/wiki/Reciprocal_distribution)
+"""
+struct LogUniform{T<:Real} <: ContinuousUnivariateDistribution
+    a::T
+    b::T
+    LogUniform{T}(a::T, b::T) where {T <: Real} = new{T}(a, b)
+end
+Base.eltype(::Type{LogUniform{T}}) where {T} = float(T)
```

Comment:
This is something that is discussed from time to time but the current policy is not let the type of the parameters determine the type of the samples:
```suggestion
```
I am currently working on a PR that would still allow to generate samples of different types (and not only `Float64`), and there I'll also list more reasons for why IMO the type of the parameters and samples should be decoupled (simple example: `Poisson` - the type of the rate and samples have to be decoupled since the first one is usually a floating point number and the latter one an integer).